twiggy/tests/expectations/garbage

```diff
@@ -0,0 +1,11 @@
+ Bytes │ Size % │ Item
```

Comment:
Can you check in the WAT source used to generate `garbage.wasm` too? That way we can easily make changes if we need to update it in the future.